src/full/Agda/Syntax/Translation/ConcreteToAbstract.hs

```diff
@@ -1502,175 +1502,248 @@ instance ToAbstract Declarations where
 
   toAbstract (Declarations ds) = niceDecls DoWarn ds toAbstract
 
-newtype LetDefs = LetDefs (List1 C.Declaration)
-newtype LetDef = LetDef NiceDeclaration
+-- | Where did these 'LetDef's come from?
+data WhereLetDef
```

Comment:
Can we call this `LetDefOrigin`?  
(The `Where` trips me up, I am thinking about `where` blocks then.)

Reply:
👍 will change it in today's bit of hackathon